src/main/resources/hudson/plugins/emailext/Messages.properties

```diff
@@ -24,6 +24,7 @@ CulpritsRecipientProvider.DisplayName=Culprits
 DevelopersRecipientProvider.DisplayName=Developers
 FailingTestSuspectsRecipientProvider.DisplayName=Suspects causing unit tests to begin failing
 FirstFailingBuildSuspectsRecipientProvider.DisplayName=Suspects causing the build to begin failing
+PreviousRecipientProvider.DisplayName=Culprits / Requestor or Developers of the previous build(s)
```

Comment:
It seems to me to be an "and" operation, not an or. All of the developers and requester will be added based on the code, so it is not an "or" it is an "and" operation.

Reply:
You are absolutely right. I corrected it.